frontend/src/AlertsManagement/AlertsList.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import Checkbox from '@material-ui/core/Checkbox';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Tooltip from '@material-ui/core/Tooltip';
+
+class AlertsList extends Component {
+  render() {
+    const alerts = this.props.alerts;
```

Comment:
jc, not super relevant to this specific PR, but could you explain more approximately how the actual Blackswan alerts would be integrated since you're doing a lot of indexing-scheme here. once you do the fetch for the actual alerts from the servlet, how would that look like, will you still be able to do these indexing mechanisms as well?